src/promptflow/promptflow/_sdk/data/executable/utils.py

```diff
@@ -58,24 +58,24 @@ def item_render_message(value, key=None):
         show_image(value)
     else:
         if key is None:
-            st.markdown(f"`{json_dumps(value)},`")
+            st.markdown(f"{json_dumps(value)},")
         else:
-            st.markdown(f"`{key}: {json_dumps(value)},`")
+            st.markdown(f"{key}: {json_dumps(value)},")
 
 
 def list_iter_render_message(message_items):
     if is_list_contains_rich_text(message_items):
-        st.markdown("`[ `")
+        st.markdown("[ ")
```

Comment:
Why changed this?

Reply:
if add `` , streamlit will regard contents as code block and show as code block format.